### DISCUSSION TITLE
No references found for: LSP identifier at point (C#)

### DISCUSSION THREAD

**Discussion by madorian** [joaotavora/eglot#1164] [General]
I just get that message for any C# file, but it works for C++.

Is there something special that needs to be added here?

**Comment by madorian** [comment]
Why has this been moved to converstion? It's not a bug; it's a conversation?

**Comment by 6b6279** [comment]
What exactly causes this error message to be emitted? Which function are you calling? `xref-find-references`? A bit more information regarding the circumstances of this message being shown would be helpful.
